.wp-env.json

```diff
@@ -1,24 +1,24 @@
 {
-  "core": null,
-  "plugins": [
-    "./plugins/optimization-detective",
-    "./plugins/auto-sizes",
-    "./plugins/dominant-color-images",
-    "./plugins/embed-optimizer",
-    "./plugins/image-prioritizer",
-    "./plugins/performance-lab",
-    "./plugins/speculation-rules",
-    "./plugins/web-worker-offloading",
-    "./plugins/webp-uploads"
-  ],
-  "env": {
-    "tests": {
-      "config": {
-        "FS_METHOD": "direct"
-      },
-      "mappings": {
-        "/wp-content/plugins/performance": "."
-      }
-    }
-  }
+	"core": null,
```

Comment:
Per https://github.com/WordPress/performance/blob/trunk/.editorconfig#L23-L25 the json config it should use two space not tab 🤔

Reply:
It seems that when running `npm run format-js`, the `.editorconfig` settings are not being applied, which is causing JSON files to use tabs instead of 2 spaces as specified in the `.editorconfig`. It appears the command is currently using the .prettierrc configuration instead.

Reply:
When both `.editorconfig` and `.prettierrc` are present, Prettier gives precedence to `.prettierrc`. You can override config for yml and json to change this behavior from the Prettier config file - https://prettier.io/docs/en/configuration.html#configuration-overrides

Reply:
Thanks. Can we then remove this `.prettierrc.js` override for yml files? We should be able to use the default config for those too, no?